drivers/hmis/app/jobs/hmis/activity_log_processor_job.rb

```diff
@@ -42,9 +42,6 @@ def perform(force: false)
             joins(:project).
             pluck(e_t[:id], p_t[:id])
```

Comment:
This could be simplified to the below, right? Since we don't need to join anymore, we don't need the `unscoped` piece either.
```
project_id_map  = Hmis::Hud::Enrollment.with_deleted.where(id: enrollment_ids).pluck(:id, :project_pk).to_h
```

Reply:
Nice! Thanks!